tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/internal/__bindingIdCamelCase__HandlerFactory.java

```diff
@@ -29,7 +29,7 @@
  *
  * @author ${author} - Initial contribution
  */
-@Component(service = ThingHandlerFactory.class, immediate = true)
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPolicy = ConfigurationPolicy.OPTIONAL, name = "binding.${bindingId}")
```

Comment:
@triller-telekom Why did you actually add `configurationPolicy = ConfigurationPolicy.OPTIONAL` in this PR? Afaik, this is the default setting, so I wonder if we should really specifically add it.